pmda/rmsf.py

```diff
@@ -0,0 +1,216 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
We should create a `rms` module as in `MDAnalysis.analysis.rms`. Instead of putting it all in one file, we could make a submodule
```
rms/__init__.py
      rmsd.py
      rmsf.py
```
and in `__init__.py` we have
```python
from __future__ import absolute_import
from .rmsf import RMSF
from .rmsd import RMSD

__all__ = ["RMSD", "RMSF"]
```

Reply:
Will also require doc restructuring...

Reply:
Did we still want to do this, or does this qualify for its own PR after parallel_rmsf is merged?

Reply:
Let's do a separate PR right after this one.

Reply:
Better to move forward in small, manageable steps than try to solve all problems in one go.